### PR TITLE
Update to support darkmode

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #102 #98

Once released, the following PR needs to be approved and shipped
https://github.com/AlaskaAirlines/AuroDocsSite/pull/145

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

This PR will deprecate legacy named tokens, e.g. onLight and onDark and replace them with the new naming convention of default and inverse.

This naming is designed to move away from onDark and not confuse this with dark mode color support.

Changes to be committed:
modified:   src/color/alert.json
modified:   src/color/border.json
modified:   src/color/icon.json
modified:   src/color/text.json
modified:   src/color/tier.json
modified:   src/color/ui.json

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/color/ui.json [hunk 1]
```diff
@@ -97,6 +146,16 @@
             "public": true,
             "onLight": true,
             "usage": "Default hover affordance for background colors",
+            "deprecated": true,
+            "comment": "Deprecated token",
+            "reference": "color-ui-bkg-hover  -default",
```
Comment (jason-capsule42) [inline src/color/ui.json:151]:
```suggestion
            "reference": "color-ui-bkg-hover-default",
```

File: src/color/ui.json [hunk 2]
```diff
@@ -97,6 +146,16 @@
...
+            "version": "3.15.0"
+          },
+          "default": {
+            "value": "0000000f",
```
Comment (jason-capsule42) [inline src/color/ui.json:155]: Is ths supposed to be a hex value?

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/color/ui.json (hunk 2 above)
Comment (blackfalcon) [reply src/color/ui.json:155]: It's an 8 digit hex. https://www.quackit.com/css/color/values/css_hex_color_notation_8_digits.cfm#:~:text=Eight%2Ddigit%20hex%20notation%20consists,alpha%20chanel%20of%20the%20color.&text=The%20RR%20represents%20the%20red,GG%20represents%20the%20green%20component.

Comment (jason-capsule42) [reply src/color/ui.json:155]: Makes sense, just wanted to make sure it wasn't a copy paste error.

**Review by jason-capsule42 — APPROVED** [review]

**Comment by blackfalcon** [comment]
This PR continues to fail. I reverted the merge and will be addressing the tokens updates here https://github.com/AlaskaAirlines/AuroDesignTokens/pull/116

The other infrastructure updates will be addressed in another PR post the release of these updates. 

- https://github.com/AlaskaAirlines/AuroDesignTokens/pull/105/commits/7e871c8ed6a745227a45589f428b126f8f0a87bb
- https://github.com/AlaskaAirlines/AuroDesignTokens/pull/105/commits/ca7488e33c1be4cd2a7616aee4a9d5ecef4b893c
- https://github.com/AlaskaAirlines/AuroDesignTokens/pull/105/commits/710fad153e2bbc5c79efdc817ab605a77221ff3d

**Pull request merged**